### PR TITLE
Added the frontend component to display products in a grid layout

### PR DESCRIPTION
**Pull request by kouamschekina**
Added the following components
- One for a single product display
- One for a list of products, reusing the first one
 each with their storybooks

 Wrote tests for the two components

### CODE REVIEW

File: .env.example
Comment (stephane-segning) [inline .env.example:1]: Why did you removed this file?

**Comment by stephane-segning** [comment]
Screenshot 2024-11-13 at 09 22 43

When you hover on the product card, it'll zoom-in. Is it normal?

**Comment by stephane-segning** [comment]
Screenshot 2024-11-13 at 09 26 11
Screenshot 2024-11-13 at 09 26 43


Ensure we increase the number of columns the wider the screen is

**Review by Valsuh45 — APPROVED** [review]
LGTM :+1:

**Review by stephane-segning — APPROVED** [review]

**Pull request merged**